docs/virtual_machines/startup_scripts.md

```diff
@@ -742,3 +750,474 @@ spec:
+          </OOBE>
+          <AutoLogon>
+            <Password>
+            <Value>123456</Value>
+              <PlainText>true</PlainText>
+            </Password>
+            <Enabled>true</Enabled>
+        <Username>Administrator</Username>
+    </AutoLogon>
+    <UserAccounts>
+         <AdministratorPassword>
+                <Value>123456</Value>
+                <PlainText>true</PlainText>
+        </AdministratorPassword>
+          </UserAccounts>
+          <RegisteredOrganization>Kuebvirt</RegisteredOrganization>
+          <RegisteredOwner>Kubevirt</RegisteredOwner>
+          <TimeZone>Eastern Standard Time</TimeZone>
+                <FirstLogonCommands>
+                    <SynchronousCommand wcm:action="add">
+                    <CommandLine>powershell -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -NoProfile d:\customize.ps1</CommandLine>
+                        <RequiresUserInput>false</RequiresUserInput>
+                        <Order>1</Order>
+                <Description>Customize Script</Description>
+            </SynchronousCommand>
+                </FirstLogonCommands>
+        </component>
+      </settings>
+    </unattend>
+  customize.ps1: |-
+    # Enable RDP
+    Set-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server' -name "fDenyTSConnections" -value 0
+    Enable-NetFirewallRule -DisplayGroup "Remote Desktop"
+
+
+    # https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse
+    # Install the OpenSSH Server
+    Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+    # Start the sshd service
+    Start-Service sshd
+
+    Set-Service -Name sshd -StartupType 'Automatic'
+
+    # https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration
+    # use powershell as default shell for ssh
+    New-ItemProperty -Path "HKLM:\SOFTWARE\OpenSSH" -Name DefaultShell -Value "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -PropertyType String -Force
+
+
+    # Add ssh authorized_key for administrator
+    # https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement
+    $MyDir = $MyInvocation.MyCommand.Path | Split-Path -Parent
+    $PublicKey = Get-Content -Path $MyDir\id_rsa.pub
+    $authrized_keys_path = $env:ProgramData + "\ssh\administrators_authorized_keys" 
+    Add-Content -Path $authrized_keys_path -Value $PublicKey
+    icacls.exe $authrized_keys_path /inheritance:r /grant "Administrators:F" /grant "SYSTEM:F"
+
+
+    # install application via exe file installer from url
+    function Install-Exe {
+      $dlurl = $args[0]
+      $installerPath = Join-Path $env:TEMP (Split-Path $dlurl -Leaf)
+      Invoke-WebRequest -UseBasicParsing $dlurl -OutFile $installerPath
+      Start-Process -FilePath $installerPath -Args "/S" -Verb RunAs -Wait
+      Remove-Item $installerPath
+
+    }
+
+    # https://superuser.com/questions/747149/
```

Comment:
Can we just change comment to
`Wait for networking before running a task at startup`